ini_python_multiline_test.go

```diff
@@ -0,0 +1,65 @@
+package ini_test
+
+import (
+	"path/filepath"
+	"testing"
+
+	"gopkg.in/ini.v1"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+type testData struct {
+	Value1 string `ini:"value1"`
+	Value2 string `ini:"value2"`
+	Value3 string `ini:"value3"`
+}
+
+func TestMultiline(t *testing.T) {
+	Convey("Parse Python-style multiline values", t, func() {
+		path := filepath.Join("testdata", "multiline.ini")
+		f, err := ini.LoadSources(ini.LoadOptions{
+			AllowPythonMultilineValues: true,
+			ReaderBufferSize: 64*1024,
+			/*
+			Debug: func(m string) {
+				fmt.Println(m)
+			},
+			 */
```

Comment:
Why is this comment necessary?

Reply:
This is what I used to debug the code change. It is also an example for someone who wants to know how to use it. It is in the test case, not the core module, so I don't think it is an issue.